CONTRIBUTING.md

```diff
@@ -54,11 +56,16 @@ $ buildah bud -t <MYREPO>/<MYIMAGE>:<MYTAG> -f Dockerfile.dev .
 $ oc patch configmap openshift-controller-manager-images -n openshift-controller-manager-operator --patch '{"data":{"builderImage":"<MYREPO>/<MYIMAGE>:<MYTAG>"}}' --type=merge
 ```
 
-6. Watch the openshift controller manager daemonset rollout (this can take a few minutes):
+6. Watch the openshift controller manager deployment/daemonset rollout (this can take a few minutes):
 
+If you are running on `4.11` or earlier use:
```

Comment:
I am tempted to say that we should remove the part about 4.11, since this doc is for 4.13+, what do you think?

Reply:
Make sense :+1: